build/tslint.json

```diff
@@ -39,19 +50,20 @@
       "trace"
     ],
     "no-construct": true,
-    "no-constructor-vars": false,
     "no-debugger": true,
     "no-duplicate-variable": true,
     "no-empty": false,
     "no-eval": true,
-    "no-inferrable-types": false,
+    "no-inferrable-types": [
+      false
+    ],
     "no-internal-module": true,
     "no-null-keyword": false,
     "no-require-imports": true,
     "no-shadowed-variable": true,
     "no-string-literal": false,
     "no-switch-case-fall-through": true,
-    "no-trailing-whitespace": true,
+    "no-trailing-whitespace": false,
```

Comment:
If not set it to false, I see errors from this kind of code.

```
throw new Error(
                `The argument must be an object literal passed in the following pattern: 
                { "shape-name": "shape-template" }`
            );
```

I also like to set this one true if team will be willing to change code.